03/app.js

```diff
@@ -42,11 +42,31 @@ books.getAuthor = function(isbn) {
 }
 
 books.getTitle = function(isbn, lang) {
+    if(typeof this[isbn] === 'undefined') {
+        return null;
+    }
+    const title = this[isbn]['title'];
+    if(typeof this[isbn]['title'][lang] === 'undefined') {
```

Comment:
zobacz, że w title masz już ten obiekt wiec wystarczyło by to wykorzystać tj. `if(typeof title[lang] === 'undefined')`